pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -323,14 +323,14 @@ func (p *PoolManager) IsKubevirtEnabled() bool {
 	return p.isKubevirt
 }
 
-func (p *PoolManager) isRelatedToKubevirt(pod *corev1.Pod) bool {
+func (p *PoolManager) isRelatedToKubevirt(pod *corev1.Pod, kubevirtObjectNamespace string) bool {
```

Comment:
This is no longer kubevirtObject right ? since we take it from Networks, I would say to name this `namespace` so we hidde where is comming from.